pal/os/inc/az_ulib_pal_os_api.h

```diff
@@ -27,37 +24,37 @@ extern "C"
  *
  * @param[in,out]   lock    The #az_ulib_pal_os_lock* that points to the lock handle.
  */
-MOCKABLE_FUNCTION(, void, az_pal_os_lock_init, az_ulib_pal_os_lock*, lock);
```

Comment:
Amen!

Reply:
:D

---
In reply to: [590630737](https://github.com/Azure/azure-ulib-c/pull/61#discussion_r590630737) [](ancestors = 590630737)